switchery.js

```diff
@@ -75,7 +75,12 @@ function Switchery(element, options) {
  */
 
 Switchery.prototype.hide = function() {
-  this.element.style.display = 'none';
+  this.element.style.height = '100%';
+  this.element.style.width = '100%';
+  this.element.style.opacity = '.0001';
+  this.element.style.position = 'absolute';
+  this.element.style.left = '0';
+  this.element.style.top = '0';
```

Comment:
ℹ️ (medium priority)

Assuming this is to make the `<input type="checkbox">` element focusable while also not visible on screen, I recommend bumping it off-screen which can be done consistently via:

```js
bottom: 100%;
overflow: hidden;
position: fixed;
right: 100%;
```

I prefer this method because it hides everything regardless of the size of the element, and doesn't accidentally introduce scrollbars due to the content always being off the top left of the screen.

Reply:
Awesome, thanks! I've committed your suggestion. 🙌